types/nightwatch/index.d.ts

```diff
@@ -1865,7 +1865,7 @@ export class Element {
     message: string;
     timeout: number;
     getId: () => string;
-    findElement: ElementCommands['findElement'];
+    findElement: ElementCommands['findElement'] & {(): Awaitable<NightwatchAPI, WebElement>};
```

Comment:
Does this also need to be fixed for `findElements`?

Reply:
No, this behaviour is specific to `findElement`.